### PR TITLE
Dex 17120

### PR DESCRIPTION
**Pull request by Matei-Iordache**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--www-landing-pages--bitdefender.hlx.page/drafts/test-page/
- After: https://DEX-17120--www-landing-pages--bitdefender.hlx.page/drafts/test-page/

### CODE REVIEW

**Comment by vradulescu-bd** [comment]
don't forget to update the snapshot tests and add in sidekick new variation

File: _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.css [hunk 1]
```diff
@@ -230,6 +231,67 @@
...
+  font: normal normal bold 18px/21px Roboto, sans-serif;
+  color: #000;
+  border-bottom: 1px solid #CCC;
+  margin-bottom: 0.8em;
+  padding-bottom: 0.6em;
+  display: block;
+  width: 92%;
+  margin: 0 auto;
+}
+.b-productswithinputdevices-container.second-template .b-productswithinputdevices > div > div p:nth-child(1) strong {
```
Comment (vradulescu-bd) [inline _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.css:265]: is this 100% always strong ? if this strong is part of content, then we shouldn't rely on this

File: _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.js [hunk 2]
```diff
@@ -186,8 +186,35 @@ export default function decorate(block) {
       if (bulinaText) {
         pricesDiv.innerHTML += `<div class="prod-percent green_bck_circle small has${bulinaText.split(',').length}txt"><b class="percent percent-${onSelectorClass}">10%</b><p>${bulinaText.split(',')[1]}</p></div>`;
       }
-      pricesDiv.innerHTML += `<p class="">${subscribeTexts}</p>`;
-      pricesDiv.innerHTML += `<b class="">${prodYears} ${prodYears > 1 ? yearsText : yearText}</b>`;
+
+      if (secondTemplate) {
```
Comment (vradulescu-bd) [inline _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.js:190]: we need to cover this in jest also

File: _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.js [hunk 3]
```diff
@@ -186,8 +186,35 @@ export default function decorate(block) {
...
+          <option value="2">${subscribeTexts} <i style="font-weight: bold; font-style: normal">2 ${yearsText}</i></option>
+          <option value="3">${subscribeTexts} <i style="font-weight: bold; font-style: normal">3 ${yearsText}</i></option>
+        `;
+        selectElement.id = 'selectYears';
+        selectorBox.appendChild(selectElement);
+        pricesDiv.appendChild(selectorBox);
+
+        // Add an event listener for the 'change' event
+        selectElement.addEventListener('change', (event) => {
+          console.log('i get here');
```
Comment (vradulescu-bd) [inline _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.js]: remove

File: _src-lp/blocks/banner/banner.css [hunk 4]
```diff
@@ -385,4 +396,21 @@
...
+    background-color: #000;
+  }
+}
+@media screen and (width >= 1400px) {
+  .section.banner-container .custom-col-xl-4 {
+    flex: 0 0 auto;
+    width: 27.33%;
+  }
+}
+@media screen and (width >= 1800px) {
```
Comment (vradulescu-bd) [inline _src-lp/blocks/banner/banner.css:412]: 1800 ? why ? which viewport case it this ? I don't think we should have this at all

File: _src-lp/blocks/banner/banner.scss [hunk 5]
```diff
@@ -435,4 +446,24 @@
...
+}
+
+@media screen and (width >= 1400px) {
+  .section.banner-container .custom-col-xl-4 {
+    flex: 0 0 auto;
+    width: 27.33%;
+  }
+}
+
+@media screen and (width >= 1800px) {
```
Comment (vradulescu-bd) [inline _src-lp/blocks/banner/banner.scss:465]: 800 ? why ? which viewport case it this ? I don't think we should have this at all

**Review by vradulescu-bd — CHANGES_REQUESTED** [review]

File: _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.js (hunk 3 above)
Comment (Matei-Iordache) [reply _src-lp/blocks/b-productswithinputdevices/b-productswithinputdevices.js]: removed

**Review by ltiseanu — APPROVED** [review]

**Review by enake — APPROVED** [review]

**Pull request merged**